data/mods/Magiclysm/monsters/zombified_monsters.json

```diff
@@ -32,5 +32,38 @@
     "death_function": [ "NORMAL" ],
     "upgrades": { "half_life": 28, "into_group": "GROUP_ZOLF_UPGRADE" },
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "KEENNOSE", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
+  },
+  {
+    "id": "mon_zorc",
+    "type": "MONSTER",
+    "name": { "str": "undead orc" },
+    "description": " This unholy amalgamation looks at you with a piercing stare so strong it scares your soul away.",
```

Comment:
```suggestion
    "description": " This unholy amalgamation looks at you with a piercing stare so strong it might scare your soul away.",
```
Amalgamation?